src/Nancy/Diagnostics/Modules/MainModule.cs

```diff
@@ -4,12 +4,15 @@ public class MainModule : DiagnosticModule
     {
         public MainModule()
         {
-            Get["/"] = async (_, __) =>
+            Get("/", _ =>

```

Comment:
I'm going to be anal now and ask, are we going to be consistent and call it `args` or `_` everywhere?